tests/test_read_range_docker.py

```diff
@@ -110,3 +110,51 @@ def test_alluxio_client(alluxio_client: AlluxioClient):
             length,
         )
     LOGGER.debug("Passed corner test cases")
+
+
+def test_etcd_alluxio_client(etcd_alluxio_client: AlluxioClient):
```

Comment:
@ssyssy can you extract the common code to another method so that we won't duplicate code
e.g.
def test_alluxio_client(AlluxioClient):
    read_range_test(AlluxioClient)
def test_etcd_alluxio_client(AlluxioClient):
    read_range_test(AlluxioClient)